tests/arrays_test_generated.h

```diff
@@ -0,0 +1,589 @@
+  return
+      (lhs.a() == rhs.a()) &&
+      (lhs.b() == rhs.b()) &&
+      (lhs.c() == rhs.c());
+}
+
+inline bool operator!=(const NestedStruct &lhs, const NestedStruct &rhs) {
+    return !(lhs == rhs);
+}
+
+
+FLATBUFFERS_MANUALLY_ALIGNED_STRUCT(4) ArrayStruct FLATBUFFERS_FINAL_CLASS {
+ private:
+  float a_;
+  int32_t b_[15];
+  int8_t c_;
+  int8_t padding0__;  int16_t padding1__;
+  NestedStruct d_[2];
+
+ public:
+  static const flatbuffers::TypeTable *MiniReflectTypeTable() {
+    return ArrayStructTypeTable();
+  }
+  ArrayStruct() {
+    memset(static_cast<void *>(this), 0, sizeof(ArrayStruct));
+  }
+  ArrayStruct(float _a, const int32_t *_b, int8_t _c, const NestedStruct *_d)
+      : a_(flatbuffers::EndianScalar(_a)),
+        c_(flatbuffers::EndianScalar(_c)),
+        padding0__(0),
+        padding1__(0) {
+    for (flatbuffers::uoffset_t i = 0; i < 15; i++)
+      reinterpret_cast<flatbuffers::Array<int32_t, 15> *>(b_)->Mutate(i, _b[i]);
+    (void)padding0__;    (void)padding1__;
+    for (flatbuffers::uoffset_t i = 0; i < 2; i++)
+      reinterpret_cast<flatbuffers::Array<NestedStruct, 2> *>(d_)->Mutate(i, _d[i]);
+  }
+  float a() const {
+    return flatbuffers::EndianScalar(a_);
+  }
+  void mutate_a(float _a) {
+    flatbuffers::WriteScalar(&a_, _a);
+  }
+  const flatbuffers::Array<int32_t, 15> *b() const {
+    return reinterpret_cast<const flatbuffers::Array<int32_t, 15> *>(b_);
+  }
+  flatbuffers::Array<int32_t, 15> *mutable_b() {
+    return reinterpret_cast<flatbuffers::Array<int32_t, 15> *>(b_);
+  }
+  int8_t c() const {
+    return flatbuffers::EndianScalar(c_);
+  }
+  void mutate_c(int8_t _c) {
+    flatbuffers::WriteScalar(&c_, _c);
+  }
+  const flatbuffers::Array<NestedStruct, 2> *d() const {
+    return reinterpret_cast<const flatbuffers::Array<NestedStruct, 2> *>(d_);
+  }
+  flatbuffers::Array<NestedStruct, 2> *mutable_d() {
+    return reinterpret_cast<flatbuffers::Array<NestedStruct, 2> *>(d_);
+  }
+};
+FLATBUFFERS_STRUCT_END(ArrayStruct, 92);
+
+inline bool operator==(const ArrayStruct &lhs, const ArrayStruct &rhs) {
+  return
+      (lhs.a() == rhs.a()) &&
+      (lhs.b() == rhs.b()) &&
+      (lhs.c() == rhs.c()) &&
+      (lhs.d() == rhs.d());
+}
+
+inline bool operator!=(const ArrayStruct &lhs, const ArrayStruct &rhs) {
+    return !(lhs == rhs);
+}
+
+
+FLATBUFFERS_MANUALLY_ALIGNED_STRUCT(8) TestStruct FLATBUFFERS_FINAL_CLASS {
+ private:
+  uint8_t a_;
+  int8_t padding0__;
+  uint16_t b_[21];
+  uint32_t c_[2];
+  int32_t padding1__;
+  double d_[3];
+  uint16_t e_;
+  int16_t padding2__;
+  NestedStruct f_;
+
+ public:
+  static const flatbuffers::TypeTable *MiniReflectTypeTable() {
+    return TestStructTypeTable();
+  }
+  TestStruct() {
+    memset(static_cast<void *>(this), 0, sizeof(TestStruct));
+  }
+  TestStruct(uint8_t _a, const uint16_t *_b, const uint32_t *_c, const double *_d, uint16_t _e, const NestedStruct &_f)
```

Comment:
PVS-Studio:
>V730 Not all members of a class are initialized inside the constructor. Consider inspecting: b_, c_, d_. arrays_test_generated.h 212

False positive?
Generated code never called in tests (I don't found calls).

Reply:
Removed `TestStruct` from the `fbs`.